### DISCUSSION TITLE
The ability to adjust the resolution to the cutout screen on Android has disappeared

### DISCUSSION THREAD

**Discussion by SenorNikicho** [ppy/osu#29452] [Q&A] [answered]
In the graphics settings, the option to adjust the resolution to the cutout screen on Android has disappeared. Is this how it should be? If so, why?

Screenshot: 
![Screenshot_20240816_164918](https://github.com/user-attachments/assets/565368f4-ee81-4fc7-ba50-f768fc0f68d9)

**Comment by bdach** [comment]
I'm not sure what you mean by "cutout screen". Please explain.

**Reply by Jinkku** [reply]
![Screenshot_20240819_064709](https://github.com/user-attachments/assets/cf4d91f8-c573-4aad-803c-48a41205f85c)
I think they meant the camera cutout option

**Reply by SenorNikicho** [reply]
> ![Screenshot_20240819_064709](https://private-user-images.githubusercontent.com/49614252/359075295-cf4d91f8-c573-4aad-803c-48a41205f85c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQwODE5NDUsIm5iZiI6MTcyNDA4MTY0NSwicGF0aCI6Ii80OTYxNDI1Mi8zNTkwNzUyOTUtY2Y0ZDkxZjgtYzU3My00YWFkLTgwM2MtNDhhNDEyMDVmODVjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE5VDE1MzQwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU0OGY0ZGE1ZjM4MTJiYjg1YjVkYTc4NGE4NTg1ZDM0MDJlZWYwNTAyNmVlMmQwMjdlYTBhZmRkYmQxMzU3ZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ATzdL1Us5JyPLVXKhk3cd1iFJCTuPcKXUFLnxcmZNbU) I think they meant the camera cutout option

Yes, that's right. I use a translator, so I apologize for mistakes 😅

**Reply by Reco1I** [reply]
It's okay, cutout it's the proper term according to Android documentation.

**Reply by Joehuu** [reply, marked as answer]
https://github.com/ppy/osu-framework/pull/6363
